libraries/helpers_image.rb

```diff
@@ -93,6 +93,12 @@ def save_image
           Docker::Image.save(repo, destination, connection)
         end
       end
+
+      def load_image
+        with_retries do
+          Docker::Image.load(source, (), connection)
```

Comment:
i believe rubocop is complaining about your use of (). changing it to {} should clear it up.